lib/services/nativescript-migration-service.ts

```diff
@@ -74,6 +74,10 @@ export class NativeScriptMigrationService implements IFrameworkMigrationService
 				return framework.displayName;
 			}
 
+			if(_.any(this.getObsoleteVersions().wait(), obsoleteVersion => obsoleteVersion === version)) {
+				this.$errors.failWithoutHelp(`Your project is using obsolete framework version - ${version}. You can still build your project, but you cannot migrate to a newer version.`);
```

Comment:
Maybe add a message how the user can remedy the situation. Something like "To use newer version of NativeScript, re-create the project from scratch. See the docs (if we have any)"